configs/detr/detr_r50_8xb2-150e_coco_class91.py

```diff
@@ -0,0 +1,48 @@
+_base_ = ['./detr_r50_8xb2-150e_coco.py']
+
+model = dict(bbox_head=dict(num_classes=91))
+
+metainfo = dict(
+    CLASSES=(None, 'person', 'bicycle', 'car', 'motorcycle', 'airplane', 'bus',
```

Comment:
why do we need to set None here? can we directly use the real name?

Reply:
We can use the real name，but it still can not work